angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.component.spec.ts

```diff
@@ -198,17 +213,47 @@ describe('SprkMastheadComponent', () => {
   it('should add aria-controls and id to narrowNav if narrowNavId is not passed', () => {
     component.isNarrowNavOpen = true;
     fixture.detectChanges();
-    narrowNavElement = fixture.nativeElement.querySelector('.sprk-c-Masthead__narrow-nav');
-    expect(narrowNavElement.getAttribute('id')).toMatch(/sprk_masthead_narrow_nav_\d/);
-    expect(hamburgerIcon.getAttribute('aria-controls')).toEqual(narrowNavElement.getAttribute('id'));
+    narrowNavElement = fixture.nativeElement.querySelector(
+      '.sprk-c-Masthead__narrow-nav',
+    );
+    expect(narrowNavElement.getAttribute('id')).toMatch(
+      /sprk_masthead_narrow_nav_\d/,
+    );
+    expect(hamburgerIcon.getAttribute('aria-controls')).toEqual(
+      narrowNavElement.getAttribute('id'),
+    );
   });
 
   it('should add correct aria-controls and id to narrowNav if narrowNavId is passed', () => {
     component.isNarrowNavOpen = true;
     component.narrowNavId = 'test_controls';
     fixture.detectChanges();
-    narrowNavElement = fixture.nativeElement.querySelector('.sprk-c-Masthead__narrow-nav');
+    narrowNavElement = fixture.nativeElement.querySelector(
+      '.sprk-c-Masthead__narrow-nav',
+    );
     expect(narrowNavElement.getAttribute('id')).toEqual('test_controls');
-    expect(hamburgerIcon.getAttribute('aria-controls')).toEqual(narrowNavElement.getAttribute('id'));
+    expect(hamburgerIcon.getAttribute('aria-controls')).toEqual(
+      narrowNavElement.getAttribute('id'),
+    );
+  });
+
+  it('should set the href link of the bigNavLink', () => {
+    component.bigNavLinks = [{ text: 'Item 1', href: '/alert' }];
+    fixture.detectChanges();
+    const secondaryNavElementItem = fixture.nativeElement.querySelector(
+      '.sprk-c-Masthead__link--big-nav',
+    );
+    expect(secondaryNavElementItem.getAttribute('href')).toEqual('/alert');
+  });
+
+  it('should set the href link of the bigNavLink when using routerLink', () => {
+    component.bigNavLinks = [{ text: 'Item 1', routerLink: '/alert-router' }];
+    fixture.detectChanges();
+    const secondaryNavElementItem = fixture.nativeElement.querySelector(
+      '.sprk-c-Masthead__link--big-nav',
+    );
+    expect(secondaryNavElementItem.getAttribute('href')).toEqual(
+      '/alert-router',
+    );
```

Comment:
Tests for more routerLink options. Do we need a test for logo-slot/logo-slot-router?